modules/standard/Sys.chpl

```diff
@@ -246,6 +246,16 @@ module Sys {
   extern proc sys_strerror(error:err_t, ref string_out:c_string):err_t;
 
   extern proc sys_readlink(path:c_string, ref string_out:c_string):err_t;
+
+  /*Check if the environment variable `name` is defined or not.
+    If `name` is defined then return 1 and update `string_out` 
+    as the value of environment variable
+    else the function returns 0.
```

Comment:
These are some minor wording suggestions, from most to least important:

* `as the value`: `to store the value`
* `else the function`: I'd change to `otherwise the function`.  "else" sounds more like pseudocode than clear English to me.
* `Check if the... or not`: `Check whether or not the...` (this is more a personal preference than anything incorrect or deep).